runtime/wasm/src/mapping.rs

```diff
@@ -140,6 +149,8 @@ impl ValidModule {
         config.interruptable(true); // For timeouts.
         config.cranelift_nan_canonicalization(true); // For NaN determinism.
         config.cranelift_opt_level(wasmtime::OptLevel::None);
+        config.max_wasm_stack(*MAX_STACK_SIZE)?;
```

Comment:
Do you think it makes a difference to `unwrap` right here? The only error this can return is [this one](https://docs.rs/wasmtime/0.27.0/src/wasmtime/config.rs.html#637), which is when the `size` parameter is `0`.

Reply:
This could be unwrapped.

Reply:
Unwrapped!